node/rpc/src/auth_api.rs

```diff
@@ -35,7 +37,8 @@ where
     B: Beacon + Send + Sync + 'static,
 {
     let ks = data.keystore.read().await;
-    let (token,) = params;
+    let (header_raw,) = params;
```

Comment:
Do we extract anything else from `params`, if not then we could just rename it to `header_raw`?

Reply:
No, it's how the JSON-RPC interface is called by Lotus; it's nested within a single array, and we destructure that from a tuple in our code.

Reply:
ah okay